java/integTest/src/test/java/glide/standalone/CommandTests.java

```diff
@@ -1032,4 +1032,16 @@ public void sort() {
                         .get());
         assertArrayEquals(namesSortedByAge, regularClient.lrange(storeKey, 0, -1).get());
     }
+
+    @SneakyThrows
+    @Test
+    public void waitTest() {
```

Comment:
Should we test for bad input such as negative numreplicas/timeout?

Reply:
added test for negative timeout